worldedit-core/src/main/java/com/sk89q/worldedit/extent/clipboard/io/ClipboardFormat.java

```diff
@@ -68,7 +70,23 @@ public interface ClipboardFormat {
      * @param file the file
      * @return true if the given file is of this format
      */
-    boolean isFormat(File file);
+    default boolean isFormat(File file) {
+        try {
+            return isFormat(new FileInputStream(file));
+        } catch (FileNotFoundException e) {
+            return false;
+        }
+    }
+
+    /**
+     * Return whether the given stream is of this format.
+     *
+     * @param inputStream The stream
+     * @return true if the given stream is of this format
+     */
+    default boolean isFormat(InputStream inputStream) {
```

Comment:
This method should not close the stream in implementors, and it should be documented that the caller is responsible for closing it.